test/v2/unit/coins/btc.js

```diff
@@ -144,25 +147,27 @@ describe('BTC:', function() {
     });
 
     it('should generate 3/3 custom chain p2shP2sh address', function() {
-      const generatedAddress = coin.generateAddress({ keychains, threshold: 3, addressType: AbstractUtxoCoin.AddressTypes.P2SH_P2WSH, chain: 20, index: 756 });
+      const addressType = Codes.UnspentTypeTcomb('p2shP2wsh');
```

Comment:
Not sure why we're testing 3-of-3 addresses here.

Reply:
Not sure either but not sure it is in scope for this PR or native segwit generally.

Reply:
Let's leave it alone at this time, but if it causes work later, we should remove it, because that is not a use case we actually offer.